stage0/src/lib.rs

```diff
@@ -314,6 +311,17 @@ pub fn rust64_start(encrypted: u64) -> ! {
         E820EntryType::DiceData,
     ));
 
+    let extra = format!(
+        "--{}={:#018x}",
```

Comment:
Kernel arguments don't look like normal flags; in addition, `--` has semantic meaning (everything after that is passed as an argument to init):
https://docs.kernel.org/admin-guide/kernel-parameters.html

Reply:
I actually want the `--` to be included in the parameter name, since that makes it easier to read in stage 1 using `clap`. Since the Linux kernel will interpret the first `--` as special I updated the logic to insert a `--` if there is not already an instance in the command-line.